libs/cmdgroup/command.go

```diff
@@ -15,6 +15,12 @@ type CommandWithGroupFlag struct {
 	flagGroups []*FlagGroup
 }
 
+func (c *CommandWithGroupFlag) RefreshFlags() {
+	for _, fg := range c.flagGroups {
+		c.cmd.Flags().AddFlagSet(fg.flagSet)
```

Comment:
Can we call this from `AddFlagGroup` directly? Seems like that would save the extra function.